static/js/actions/email.js

```diff
@@ -31,6 +31,23 @@ export const sendEmailSuccess = createAction(SEND_EMAIL_SUCCESS);
 export const SEND_EMAIL_FAILURE = 'SEND_EMAIL_FAILURE';
 export const sendEmailFailure = createAction(SEND_EMAIL_FAILURE);
 
+export function sendEmail(
+  emailType: string,
```

Comment:
Is `emailType` an arbitrary string, or does it have to be one of a short list of strings? If the latter, please specify that list in the Flow type.